spring-integration-redis/src/main/java/org/springframework/integration/redis/inbound/RedisQueueMessageDrivenEndpoint.java

```diff
@@ -188,7 +200,12 @@ private void popMessageAndSend() {
 
 		byte[] value = null;
 		try {
-			value = this.boundListOperations.rightPop(this.receiveTimeout, TimeUnit.MILLISECONDS);
+			if (readFromLeft) {
```

Comment:
Any property references should be prepended with `this.`: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style